File: Dart/1020.dart
import 'dart:io';

void main() {
  int days = int.parse(stdin.readLineSync());
  int years = days ~/ 365;
  days = days % 365;
  int months = days ~/ 30;
  days = days % 30;

  print("$years ano(s)");
  print("$months mes(es)");
  print("$days dia(s)");
}

File: JavaScript/1020.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var days = parseInt(lines.shift());

var years = Math.floor(days / 365);
var months = Math.floor((days % 365) / 30);
var days = Math.floor((days % 365) % 30);

console.log(`${years} ano(s)`);
console.log(`${months} mes(es)`);
console.log(`${days} dia(s)`);
